Fix Could not find artifact fr.fifiz.socle.java:java-parent:pom:8.0.0-SNAPSHOT

```sh
Get groupId
2s
##[error]Process completed with exit code 1.
Run mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
  mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.242/x64
    JAVA_HOME_8.0.242_x64: /opt/hostedtoolcache/jdk/8.0.242/x64
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for fr.fifiz.training.app.java:simple-java8-validator:1.0.0-SNAPSHOT: Could not find artifact fr.fifiz.socle.java:java-parent:pom:8.0.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 5, column 11
 @
```

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -35,10 +35,15 @@ jobs:
         uses: actions/checkout@v2
       - name: Cache Maven packages
         uses: actions/cache@v1
+        env:
+          cache-name: cache-maven-dependencies
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-m2-${{ hashFiles('pom.xml') }}
-          restore-keys: ${{ runner.os }}-m2
+          key: ${{ runner.os }}-m2-${{ env.cache-name }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-m2-${{ env.cache-name }}-
+            ${{ runner.os }}-m2-
+            ${{ runner.os }}-
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
         env:

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -33,6 +33,12 @@ jobs:
         run: mvn -version
       - name: Check out from version control
         uses: actions/checkout@v2
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-m2-${{ hashFiles('pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
         env:

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -33,17 +33,6 @@ jobs:
         run: mvn -version
       - name: Check out from version control
         uses: actions/checkout@v2
-      - name: Cache Maven packages
-        uses: actions/cache@v1
-        env:
-          cache-name: cache-maven-dependencies
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-m2-${{ env.cache-name }}-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-m2-${{ env.cache-name }}-
-            ${{ runner.os }}-m2-
-            ${{ runner.os }}-
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
         env:

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -49,10 +49,16 @@ jobs:
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.version' -q -DforceStdout
       - name: Build with Maven
         run: mvn -U clean compile --file pom.xml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Unit Test with Maven
         run: mvn -U -Dmaven.main.skip test --file pom.xml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Verify with Maven
         run: mvn -U -Dmaven.main.skip -Dunit.test.skip verify --file pom.xml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       #TODO - name: Set Leak Period [SonarQube]
       #run: checkout du parent commun
       #run: mvn -U clean verify '-Dmaven.test.failure.ignore=true'

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -7,20 +7,20 @@ jobs:
     name: Get GitHub and Context informations
     runs-on: ubuntu-18.04
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
-    # for debugging
-    - name: Dump env
-      run: |
-        env | sort
-    - name: Dump GitHub context
-      env:
-        GITHUB_CONTEXT: ${{ toJson(github) }}
-      run: echo "$GITHUB_CONTEXT"
-    - name: Dump job context
-      env:
-        JOB_CONTEXT: ${{ toJson(job) }}
-      run: echo "$JOB_CONTEXT"
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # for debugging
+      - name: Dump env
+        run: |
+          env | sort
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - name: Dump job context
+        env:
+          JOB_CONTEXT: ${{ toJson(job) }}
+        run: echo "$JOB_CONTEXT"
 
   build-test-package-deploy-maven:
     runs-on: ubuntu-latest
@@ -35,6 +35,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Get artifactId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.artifactId' -q -DforceStdout
       - name: Get version
@@ -46,11 +48,11 @@ jobs:
       - name: Verify with Maven
         run: mvn -U -Dmaven.main.skip -Dunit.test.skip verify --file pom.xml
       #TODO - name: Set Leak Period [SonarQube]
-        #run: checkout du parent commun
-        #run: mvn -U clean verify '-Dmaven.test.failure.ignore=true'
-        #run: mvn -U '-Dsonar.links.scm=http://gitxxx' '-Dsonar.links.ci=http://jenkinsxxx' org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar '-Dsonar.projectKey=[groupId]-[artifactId]' '-Dsonar.branch.name=develop' '-Dsonar.projectName=[artifactId]r' '-Dsonar.buildbreaker.skip=true' '-Dsonar.projectVersion=#d0b75832'
+      #run: checkout du parent commun
+      #run: mvn -U clean verify '-Dmaven.test.failure.ignore=true'
+      #run: mvn -U '-Dsonar.links.scm=http://gitxxx' '-Dsonar.links.ci=http://jenkinsxxx' org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar '-Dsonar.projectKey=[groupId]-[artifactId]' '-Dsonar.branch.name=develop' '-Dsonar.projectName=[artifactId]r' '-Dsonar.buildbreaker.skip=true' '-Dsonar.projectVersion=#d0b75832'
       #TODO - name: Quality Gate [SonarQube]
-        #run: mvn -U '-Dsonar.links.scm=http://gitxxx' '-Dsonar.links.ci=http://jenkinsxxx' org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar '-Dsonar.projectKey=[groupId]-[artifactId]' '-Dsonar.branch.name=develop' '-Dsonar.projectName=[artifactId]' '-Dsonar.gitlab.ref_name=b7c32ea3401d64f97eecc4dd46de331b00391cd7' '-Dsonar.gitlab.commit_sha=b7c32ea3401d64f97eecc4dd46de331b00391cd7' '-Dsonar.gitlab.project_id=3555'
+      #run: mvn -U '-Dsonar.links.scm=http://gitxxx' '-Dsonar.links.ci=http://jenkinsxxx' org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar '-Dsonar.projectKey=[groupId]-[artifactId]' '-Dsonar.branch.name=develop' '-Dsonar.projectName=[artifactId]' '-Dsonar.gitlab.ref_name=b7c32ea3401d64f97eecc4dd46de331b00391cd7' '-Dsonar.gitlab.commit_sha=b7c32ea3401d64f97eecc4dd46de331b00391cd7' '-Dsonar.gitlab.project_id=3555'
       - name: Publish to GitHub Packages Apache Maven
         run: mvn -U -Dmaven.main.skip -Dunit.test.skip -Dintegration.test.skip deploy --file pom.xml
         env:

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -35,6 +35,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Get artifactId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.artifactId' -q -DforceStdout
       - name: Get version

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -35,8 +35,6 @@ jobs:
         uses: actions/checkout@v2
       - name: Get groupId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.groupId' -q -DforceStdout
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Get artifactId
         run: mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate '-Dexpression=project.artifactId' -q -DforceStdout
       - name: Get version

--- a/.github/workflows/maven-java-CI.yml
+++ b/.github/workflows/maven-java-CI.yml
@@ -3,9 +3,9 @@ name: Maven CI for Java
 on: [push]
 
 jobs:
-  get-context-info:
+  gihub-context-get-info:
     name: Get GitHub and Context informations
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
@@ -22,7 +22,8 @@ jobs:
           JOB_CONTEXT: ${{ toJson(job) }}
         run: echo "$JOB_CONTEXT"
 
-  build-test-package-deploy-maven:
+  maven-build-test-package-deploy-site:
+    name: Maven test, package and deploy Java project
     runs-on: ubuntu-latest
     steps:
       - name: Set up JDK 1.8

--- a/pom.xml
+++ b/pom.xml
@@ -24,6 +24,21 @@
     <tag>HEAD</tag>
   </scm>
 
+  <!-- Repositories nécessaire pour avoir accès aux packages GitHub -->
+  <repositories>
+    <repository>
+      <id>github</id>
+      <name>GitHub bdelion/java-parent Apache Maven Packages</name>
+      <url>https://maven.pkg.github.com/bdelion/java-parent</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
   <properties>
     <!-- Assembly -->
     <maven-assembly-plugin-skip>false</maven-assembly-plugin-skip>

--- a/pom.xml
+++ b/pom.xml
@@ -24,6 +24,21 @@
     <tag>HEAD</tag>
   </scm>
 
+  <!-- Repositories nécessaire pour avoir accès aux packages GitHub -->
+  <repositories>
+      <repository>
+          <id>github-bdelion</id>
+          <name>GitHub bdelion Apache Maven Packages</name>
+          <url>https://maven.pkg.github.com/bdelion</url>
+          <releases>
+              <enabled>true</enabled>
+          </releases>
+          <snapshots>
+              <enabled>true</enabled>
+          </snapshots>
+      </repository>
+  </repositories>
+
   <properties>
     <!-- Assembly -->
     <maven-assembly-plugin-skip>false</maven-assembly-plugin-skip>

--- a/pom.xml
+++ b/pom.xml
@@ -27,7 +27,7 @@
   <!-- Repositories nécessaire pour avoir accès aux packages GitHub -->
   <repositories>
       <repository>
-          <id>github-bdelion-java-parent</id>
+          <id>github</id>
           <name>GitHub bdelion/java-parent Apache Maven Packages</name>
           <url>https://maven.pkg.github.com/bdelion/java-parent</url>
           <releases>

--- a/pom.xml
+++ b/pom.xml
@@ -27,9 +27,9 @@
   <!-- Repositories nécessaire pour avoir accès aux packages GitHub -->
   <repositories>
       <repository>
-          <id>github-bdelion</id>
-          <name>GitHub bdelion Apache Maven Packages</name>
-          <url>https://maven.pkg.github.com/bdelion</url>
+          <id>github-bdelion-java-parent</id>
+          <name>GitHub bdelion/java-parent Apache Maven Packages</name>
+          <url>https://maven.pkg.github.com/bdelion/java-parent</url>
           <releases>
               <enabled>true</enabled>
           </releases>

--- a/pom.xml
+++ b/pom.xml
@@ -24,21 +24,6 @@
     <tag>HEAD</tag>
   </scm>
 
-  <!-- Repositories nécessaire pour avoir accès aux packages GitHub -->
-  <repositories>
-    <repository>
-      <id>github</id>
-      <name>GitHub bdelion/java-parent Apache Maven Packages</name>
-      <url>https://maven.pkg.github.com/bdelion/java-parent</url>
-      <releases>
-        <enabled>true</enabled>
-      </releases>
-      <snapshots>
-        <enabled>true</enabled>
-      </snapshots>
-    </repository>
-  </repositories>
-
   <properties>
     <!-- Assembly -->
     <maven-assembly-plugin-skip>false</maven-assembly-plugin-skip>

